domdiv/draw.py

```diff
@@ -25,19 +26,21 @@ def __init__(self):
         self.canvas = None
 
     def registerFonts(self):
-        try:
-            dirn = os.path.join(self.options.data_path, 'fonts')
+        dirn = os.path.join(self.options.data_path, 'fonts')
+        regularMpath = os.path.join(dirn, 'MinionPro-Regular.ttf')
+        boldMpath = os.path.join(dirn, 'MinionPro-Bold.ttf')
+        obliqueMpath = os.path.join(dirn, 'MinionPro-Oblique.ttf')
```

Comment:
Ah ok. Not for this PR, but I think we should integrate a font copy/download and conversion script, if only to guarantee the right names. Pretty sure we're ok copying/downloading them given that they're available. I checked the license, and we're definitely not allowed to distribute them ourselves.